guides/common/modules/proc_using-red-hat-insights.adoc

```diff
@@ -57,6 +57,20 @@ You can manually synchronize the recommendations using the following procedure:
 
 If you have not set up the API token, you are prompted to create one before using this page.
 
+.Removing hosts from Inventory of Red{nbsp}Hat Insights when they are removed from {Project}
+When hosts are removed from {Project}, they can also be removed from the inventory of Red{nbsp}Hat Insights, either automatically or manually.
+Use the following procedure to configure it automatically.
+
+.Prerequisite
+* You must have administrator privilege for the organization to enable this setting.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Configure* > *Inventory Upload*.
+. Enable the *Automatic Mismatch Deletion* setting.
+
+The hosts are automatically removed from the Insights Inventory during the next synchronization with {Project} that occurs daily by default.
+If the setting is disabled, you can still delete the hosts from the Inventory manually.
```

Comment:
>  you can still delete the hosts from the Inventory manually

Maybe something a bit more specific: you can delete the hosts from the Inventory from `access.redhat.com`?

Reply:
Addressed :+1: